apps/site/src/app/(home)/page.tsx

```diff
@@ -1,5 +1,12 @@
-import { Landing } from "./sections";
+import { About, Landing, Sponsors } from "./sections";
 
 export default function Home() {
-	return <Landing />;
+	return (
+		<>
+			<Landing />
+			<About />
+			{/* @ts-expect-error Async Server Component */}
+			<Sponsors />
```

Comment:
Issue: these sections will be shown when merging to `main` which isn't necessarily what we want until the entire home page is ready. I think it probably would have been better to resolve the merge conflicts.